tests/distributed/test_pipeline_parallel.py

```diff
@@ -0,0 +1,183 @@
+import os
+import subprocess
+import sys
+import time
+
+import openai  # use the official client for correctness check
+import pytest
+# using Ray for overall ease of process management, parallel requests,
+# and debugging.
+import ray
+import requests
+
+# downloading lora to test lora requests
+
+MAX_SERVER_START_WAIT_S = 600  # wait for server to start for 60 seconds
+# any model with a chat template should work here
+MODEL_NAME = "meta-llama/Meta-Llama-3-8B"
+EAGER_MODE = os.getenv("EAGER_MODE", None)
+CHUNKED_PREFILL = os.getenv("CHUNKED_PREFILL", None)
```

Comment:
```suggestion
CHUNKED_PREFILL = bool(int(os.getenv("CHUNKED_PREFILL", None)))
```

If we don't do this way, CHUNKED_PREFILL=0 won't work

Reply:
Good catch! Thanks!

Reply:
Done